jupyterhub/templates/hub/configmap.yaml

```diff
@@ -221,12 +221,8 @@ data:
     {{- end }}
   {{- end }}
 
-  {{- $_ := merge (dict "podKind" "user") . }}
-  {{- include "jupyterhub.prepareScope" $_ }}
-  {{- if $_.hasTolerations }}
   singleuser.tolerations: |
-    {{- $_.tolerationsList | nindent 4 }}
-  {{- end }}
+    {{- include "jupyterhub.userTolerations" | nindent 4 }}
```

Comment:
We can assume userTolerations has content, we cant with the affinity later though...